dev/TEAL.md

```diff
@@ -49,6 +58,30 @@ Constants are loaded into the environment by two opcodes, `intcblock` and `bytec
 
 Constants are pushed onto the stack by `intc`, `intc_[0123]`, `bytec`, and `bytec_[0123]`. The assembler will handle converting `int N` or `byte N` into the appropriate form of the instruction needed.
 
+### Named Integer Constants
+
+#### OnComplete
```

Comment:
Needs a preamble (~2 sentences) that says something about what these constants represents. Something like this: "An application transaction must indicate the action to be taken following the execution of its approvalProgram or clearStateProgram. The constants below describe the available actions."